site/content/kapp-controller/docs/latest/install.md

```diff
@@ -15,3 +15,41 @@ or
 ```bash
 $ kubectl apply -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/latest/download/release.yml
 ```
+
+## Specific Environments and Distributions
+Some kubernetes distributions require specific setup.
+Notes below capture the wisdom of our collective community - we
+appreciate your corrections and contributions to help everyone install
+kapp-controller everywhere.
+
+### Openshift
+1. Explicitly set resource packageinstalls/finalizers for kapp controller cluster role to access (else the kapp controller fails to create packageinstalls).
+```
+kind: ClusterRole
+metadata:
+  name: kapp-controller-cluster-role
+rules:
+- apiGroups:
+  - packaging.carvel.dev
+  resources:
+  ...
+  - packageinstalls/finalizers
+```
+2. Bind the kapp controller cluster role to a security context constraint that allows uids/gids that kapp deployment uses.
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kapp-controller-cluster-role
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - nonroot
```

Comment:
This `nonroot` security context constraint was what I had in my testing environment. I think it's good to call out this is configurable based on users' own Openshift environment (as long as it allows kapp uid `1000`).